app/uk/gov/hmrc/nisp/views/excluded_sp_old.scala.html

```diff
@@ -37,10 +37,14 @@
     @main(browserTitle=Some(Messages("nisp.main.title")), userLoggedIn = true,
         pageTitle = Some(Messages("nisp.excluded.title")), analyticsAdditionalJs = Some(analyticsAdditionalJs)) {
 
+        @if(spExclusions.exclusions.contains(Exclusion.ManualCorrespondenceIndicator)) {
+            @exclusions.dead()
+            ???
```

Comment:
This was my original code when I was trying to fit the logic. I wanted to display an error message so I could test it, but I never got that far,

Also the `???` means I don't know what to write next